### PR TITLE
Added Complete Story

### PR DESCRIPTION
**Pull request by heykal-crs**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,5 +4,5 @@ Commit 1<br>
 Commit 2<br>
 Commit 3
 
-I've been looking for **pizza** for several hours.
+I've been looking for **pizza** for several hours. And i was waiting too long
```
Comment (sayid-alt) [inline README.md]: You must add <br> after word "hours", so you write it down on the new line. thanks

**Review by sayid-alt — CHANGES_REQUESTED** [review]
Please change and correct the code as we suggested as well

File: README.md (hunk 1 above)
Comment (heykal-crs) [reply README.md]: Hello, thanks for reviewing my code, i've changed my code as you suggested.

Comment (sayid-alt) [reply README.md]: Thanks for contributing on this repo, you did the great work today. I appreciate it.

**Review by sayid-alt — APPROVED** [review]
Changing is accepted, thanks for contributing in this repo, you did the great works today, i appreciate it.

**Pull request merged**